bundles/org.openhab.binding.insteon/src/main/resources/OH-INF/config/config.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="channel-type:insteon:button">
+		<parameter name="related" type="text">
+			<label>Related Devices</label>
+			<description>List of related Insteon devices separated by a '+' sign.</description>
```

Comment:
Why use a "+" sign as the separator? Also, why not set parameter.multiple=true? I think that it automatically uses a comma to separate the values though, so if '+' is required to be your separate I guess you have to keep it as is.

Reply:
It's required, see https://www.openhab.org/addons/bindings/insteon/#channel-related-property